rosetta/services/block.go

```diff
@@ -585,27 +586,33 @@ type blockSignerInfo struct {
 
 // TransactionMetadata ..
 type TransactionMetadata struct {
+	// CrossShardIdentifier is the transaction identifier on the from/source shard
 	CrossShardIdentifier *types.TransactionIdentifier `json:"cross_shard_transaction_identifier,omitempty"`
-	ToShardID            uint32                       `json:"to_shard,omitempty"`
-	FromShardID          uint32                       `json:"from_shard,omitempty"`
-	Data                 string                       `json:"data,omitempty"`
+	ToShardID            *uint32                      `json:"to_shard,omitempty"`
```

Comment:
why change to pointers?

Reply:
The shards are optional for the metadata and I don't want to unmarshal it to 0 if it's not present since that is a valid shard ID. 

https://play.golang.org/p/9yNLRRwdFw7